tutorials/dashboards/README.md

```diff
@@ -49,7 +49,7 @@ Think of FormHandler as 'SQL over HTTP'. If [data](../../quickstart/data)
 is our typical data endpoint, then [data?Segment=Consumer](../../quickstart/data?Segment=Consumer)
 returns only those rows which have "Consumer" in the "Segment" column.
 Check out the list of possible operations in the
-[formhandler documentation](/formhandler/#formhandler-filters).
+[formhandler documentation](guide/formhandler/#formhandler-filters).
 
```

Comment:
@Bharath4259 -- Relative URL would be nice, I'll patch this with `../../formhandler/#formhandler-filters`. Thanks for this.